assets/scripts/info_bubble/InfoBubble.jsx

```diff
@@ -134,12 +160,17 @@ class InfoBubble extends React.Component {
     }
 
     const height = bubbleHeight + 30
-
     infoBubble.bubbleHeight = bubbleHeight
 
     this.el.style.webkitTransformOrigin = '50% ' + height + 'px'
     this.el.style.MozTransformOrigin = '50% ' + height + 'px'
     this.el.style.transformOrigin = '50% ' + height + 'px'
+
+    if (this.state.type === INFO_BUBBLE_TYPE_RIGHT_BUILDING) {
```

Comment:
I recommend adding your notes about why we do the recalculation for the `left` position here when the info bubble is activated for the right building, so we remind ourselves for the future!